.goreleaser.yml

```diff
@@ -23,13 +23,3 @@ archive:
       format: zip
   files:
     - LICENSE
-
-dockers:
-  -
-    goos: linux
-    goarch: amd64
-    binaries:
-      - secrethub-http-proxy
-    image_templates:
-      - "secrethub/http-proxy:{{ .Version }}"
-      - "secrethub/http-proxy:latest"
```

Comment:
Did this automatically push the built docker image to the Docker registry or did we have to do that ourselves? Reason I'm asking is if it did, we should add that functionality back in somewhere.

Reply:
Yes, goreleaser would use the locally configured docker client to push the image to the registry. I've now set up automated builds on DockerHub to do this for us.